### ISSUE TITLE
Bogus ResolvePackageNotFound

### ISSUE THREAD

**Issue by pvanderlinden** [conda/conda#6679]
I'm trying to install a package from our internal conda channel, but since 4.4 conda fails to do this if you do it with an environment file, and I'm not sure what causes this. Downgrading conda solves the issue, but brings other issues.

With conda 4.3 I can install the package from our internal conda channel just fine in any way. With conda 4.4 I can only install the package with `conda install`, but not with `conda env update/create` with an environment.yml file, you then get a `ResolvePackageNotFound`. 

```
>> conda info

     active environment : None
       user config file : /root/.condarc
 populated config files : /root/.condarc
          conda version : 4.4.6
    conda-build version : not installed
         python version : 3.5.4.final.0
       base environment : /opt/conda  (writable)
           channel URLs : http://internal/pkgs/linux-64
                          http://internal/pkgs/noarch
                          https://repo.continuum.io/pkgs/main/linux-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
          package cache : /opt/conda/pkgs
                          /root/.conda/pkgs
       envs directories : /opt/conda/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.4.6 requests/2.18.4 CPython/3.5.4 Linux/4.4.0-104-generic debian/8 glibc/2.19
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```

Tested in the continuumio/miniconda3:latest image, after:
```
conda install python=3.5
conda update conda
```

**Comment by pvanderlinden** [comment]
This seems to be caused by having it in a subdirectory called '/pkgs/'

**Comment by kalefranz** [comment]
I think this is probably solved with https://github.com/conda/conda/pull/6746.  Will be included in conda 4.4.8.
